README.md

```diff
@@ -182,6 +180,7 @@ cnquery bundle upload mypack.mql.yaml
 | Microsoft Azure subscriptions | `azure`                    | `cnquery shell azure --subscription SUBSCRIPTION_ID`                                                                                                        |
 | Microsoft Azure instances     | `ssh`                      | `cnquery shell ssh user@host`                                                                                                                               |
 | Okta                          | `okta`                     | `cnquery shell okta --token TOKEN --organization ORGANIZATION`                                                                                              |
+| Oracle Cloud Infrastructure (OCI) | `oci`                     | `cnquery shell oci`                                                                                              |
```

Comment:
This needs some fmt

Reply:
We're also missing a few from this list:

  arista           Scan an Arista endpoint.
  opcua            Scan an OPC UA endpoint.